pythoncalculator/divide.py

```diff
@@ -0,0 +1,2 @@
+def divide(x, z):
+    return x / y
```

Comment:
you should divide variables provided to the function, i.e. 'x/y', or supply variables in function 'x,y'.